.circleci/config.yml

```diff
@@ -67,9 +67,29 @@ jobs:
           path: reports
           destination: reports
 
+  code_coverage:
+    working_directory: ~/project
+    docker:
+      - image: circleci/android:api-28
+    environment:
+      JVM_OPTS: -Xmx1024m
+      GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError"'
+      TERM: dumb
+    steps:
+      - checkout
+      #- *restore_gradle_cache
+
+      # Run JaCoCo Reports & Upload to codecov
+      - run:
+          name: Run Code Coverage
+          command: |
+            ./gradlew jacocoTestReport -PauthToken=$JITPACK_AUTH_TOKEN
```

Comment:
Since `jacocoTestReport` is a Gradle task, would probably be a lot faster to just add it to the `assemble` Gradle execution above (so we don't need to spin up another fresh Docker image).

Reply:
faster in compute hours, not faster in absolute time